pkg/api/v1alpha1/types.go

```diff
@@ -44,13 +44,22 @@ type DeschedulerStrategy struct {
 	Params *StrategyParameters `json:"params,omitempty"`
 }
 
+// Namespaces carries a list of included/excluded namespaces
+// for which a given strategy is applicable.
+// Included namespace takes preference over excluded namespaces (they are ignored).
```

Comment:
again, probably just clearer to note that only one of these can be set and return an error rather than taking implicit preference

Reply:
Yeah, I was thinking about introducing validation for strategy params